src/battery-graph.in

```diff
@@ -153,13 +160,66 @@ adjustment=$(( $adjustment - $utc_adjust ))
 from2000=$(( $from - $adjustment ))
 to2000=$(( $to - $adjustment ))
 
+RANDOM_EXTENSION=`tr -cd '[:alnum:]' < /dev/urandom | fold -w8 | head -n1`
+TMPFILENAME="$TMPDIR/battery-graph.tmp-$RANDOM_EXTENSION"
```

Comment:
Good point, will o.

On 04/14/2016 10:09 PM, petterreinholdtsen wrote:
> In src/battery-graph.in
> <https://github.com/petterreinholdtsen/battery-stats/pull/16#discussion_r59827592>:
> 
>> @@ -153,13 +160,66 @@ adjustment=$(( $adjustment - $utc_adjust ))
>>  from2000=$(( $from - $adjustment ))
>>  to2000=$(( $to - $adjustment ))
>>  
>> +RANDOM_EXTENSION=`tr -cd '[:alnum:]' < /dev/urandom | fold -w8 | head -n1`
>> +TMPFILENAME="$TMPDIR/battery-graph.tmp-$RANDOM_EXTENSION"
> 
> Please use mktemp instead of your own method to come up with random names.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/petterreinholdtsen/battery-stats/pull/16/files/02882dc26a20a26ebe0221937bfe6ae4e57a709a#r59827592>
>